renderdoc/driver/vulkan/vk_pixelhistory.cpp

```diff
@@ -903,7 +919,40 @@ struct TestsFailedCallback : public VulkanPixelHistoryCallback
   uint32_t GetEventFlags(uint32_t eventId)
   {
     auto it = m_EventFlags.find(eventId);
-    RDCASSERT(it != m_EventFlags.end());
+    if(it == m_EventFlags.end())
+      RDCERR("Can't find event flags for event %u", eventId);
+    return it->second;
+  }
+
+  void FetchOcclusionResults()
+  {
+    if(m_OcclusionQueries.empty())
+      return;
+    m_OcclusionResults.resize(m_OcclusionQueries.size());
+    VkResult vkr =
+        ObjDisp(m_pDriver->GetDev())
+            ->GetQueryPoolResults(
+                Unwrap(m_pDriver->GetDev()), m_OcclusionPool, 0, (uint32_t)m_OcclusionResults.size(),
+                m_OcclusionResults.size() * sizeof(m_OcclusionResults[0]), m_OcclusionResults.data(),
```

Comment:
Completely unimportant but you might find it useful - I added a `byteSize()` member of `rdcarray` to do this common `rdcarray.size() * sizeof(element)` operation.

Reply:
Oh nice, done, also changed in other places.